src/Controller/HomeController.php

```diff
@@ -10,10 +10,33 @@ class HomeController extends AbstractController
     /**
      * @Route("/", name="home")
      */
+
     public function index()
     {
+
+        $data= file_get_contents('https://hw1.nfq2019.online/students.json');
+        $obj = json_decode($data);
+
+        $result = [];
+        foreach ($obj as $key=>$students) {
```

Comment:
Yra gera praktika naudoti kodo stilių.
Ar bent jau susižiūrėti `PHPStorm`, `vim`, `sublime` ar kito redaktoriaus automatinį suformatavimą.

https://github.com/aurelijusb/kickstart/pull/247/checks?check_run_id=316784991
![image](https://user-images.githubusercontent.com/1453957/69482020-edd21f80-0e1e-11ea-8ece-22b29821ba50.png)

Plačiau:
 * https://medium.com/@haydar_ai/why-we-should-use-proper-code-styles-9ace845cc4f3
 * https://github.com/squizlabs/PHP_CodeSniffer
 * https://github.com/squizlabs/PHP_CodeSniffer/wiki/Fixing-Errors-Automatically
 * https://www.jetbrains.com/help/phpstorm/configuring-code-style.html
 * https://github.com/nfqakademija/kickstart/blob/master/phpcs.xml.dist#L11